packages/api-server/tests/www.ts

```diff
@@ -0,0 +1,15 @@
+const jayson = require("jayson/promise");
+
+export const client = jayson.Client.http({
```

Comment:
Why not use `RPC` as project used?

Reply:
Do you mean [`godwoken::RPC`](https://github.com/nervosnetwork/godwoken-web3/blob/main/packages/godwoken/src/rpc.ts#L12-L23)?
I just followed [the previous code](https://github.com/nervosnetwork/godwoken-web3/pull/254/files#diff-d76812ac178d35280a797572cf21c8d2736bd4be013cb1c5341a1fc3b6fea4fcL8-L10). Do you think it's better to use [`godwoken::RPC`](https://github.com/nervosnetwork/godwoken-web3/blob/main/packages/godwoken/src/rpc.ts#L12-L23)? If so, we can do it in another PR.

Reply:
I mean `import { RPC } from "@ckb-lumos/toolkit"`, I just want to know why you use jayson client instead